ePBFT/src/pbft/libbyz/New_key.cpp

```diff
@@ -79,6 +79,22 @@ bool New_key::verify()
   {
     // Received message from unknown sender
     LOG_INFO << "Request from unknown pricipal, id:" << id() << std::endl;
+
+    std::string pubk_sig =
```

Comment:
Why is this necessary?

Reply:
This is a layover from stand-alone pbft. It will not be needed when we have the integration fully complete and can test ePBFT just inside of CCF